README.md

```diff
@@ -45,7 +45,7 @@ This project was deployed on github pages. [This is the link](https://carlylechi
 
 ### Documentation
 
-I have recorded [a loom video](https://www.loom.com/share/8ac3176a9f0a4cf0b0cadfaacb2ca02b) briefly throwing some light on the project.
+I have recorded [a loom video](https://www.loom.com/share/8ac3176a9f0a4cf0b0cadfaacb2ca02b) briefly throwing some light on the project. Feel free to check it out.
```

Comment:
- [ ] [REQUIRED] Kindly fix the live version as the link seems broken. [Live Version](https://carlylechia.github.io/Vision-Straight/index.html)
It is important to deploy tour projects in order for other users to have access to it online.